connector/src/main/java/org/jboss/as/connector/subsystems/datasources/AbstractDataSourceService.java

```diff
@@ -187,22 +187,19 @@ protected synchronized void stopService() {
                 BootstrapContextCoordinator.getInstance().removeBootstrapContext(deploymentMD.getBootstrapContextIdentifier());
             }
 
-
-        }
-
-        if (deploymentMD.getDataSources() != null && managementRepositoryValue.getValue() != null) {
-            for (org.jboss.jca.core.api.management.DataSource mgtDs : deploymentMD.getDataSources()) {
-                managementRepositoryValue.getValue().getDataSources().remove(mgtDs);
+            if (deploymentMD.getDataSources() != null && managementRepositoryValue.getValue() != null) {
+                for (org.jboss.jca.core.api.management.DataSource mgtDs : deploymentMD.getDataSources()) {
+                    managementRepositoryValue.getValue().getDataSources().remove(mgtDs);
+                }
             }
-        }
-
-        if (deploymentMD.getConnectionManagers() != null) {
-            for (ConnectionManager cm : deploymentMD.getConnectionManagers()) {
-                cm.shutdown();
+    
```

Comment:
You should see the same build errors locally if you do a "./build.sh clean install" from the root WildFly folder.  Seems to be a check-style complaint about line 195:
AbstractDataSourceService.java:195: Line has trailing spaces.